clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2450,6 +2455,7 @@ void pollStatusUntilComplete(
           VenicePushJob.this.pushJobSetting.storeResponse.getStore().getBootstrapToOnlineTimeoutInHours();
       long durationMs = System.currentTimeMillis() - pollStartTimeMs;
       if (durationMs > TimeUnit.HOURS.toMillis(bootstrapToOnlineTimeoutInHours)) {
+        sendPushJobDetailsToController();
```

Comment:
Can you explain a bit about how this will help? IIUC, if an exception is thrown, the catch clause will send push job details after marking it as an ERROR